src/screens/FeedScreen/Content.js

```diff
@@ -235,7 +237,7 @@ const Content = ({
     <Pressable
       onLayout={hanldeHeightContainer}
       onPress={onPress}
-      style={[styles.contentFeed, style]}>
+      style={[styles.contentFeed(hasComment), style]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style of the `Pressable` component is now conditionally set based on the `hasComment` prop. This change seems fine as long as the `hasComment` prop is always passed correctly.
```diff
+  style={[styles.contentFeed(hasComment), style]}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->